src/components/Reports/SalaryCalculator/SalaryCalculatorTestWrapper.tsx

```diff
@@ -19,8 +20,22 @@ export const SalaryCalculatorTestWrapper: React.FC<
     <TestRouter>
       <GqlMockedProvider<{
         Hcm: HcmQuery;
+        PayrollDates: PayrollDatesQuery;
       }>
         mocks={{
+          PayrollDates: {
+            payrollDates: [
+              {
+                regularProcessDate: '2025-01-15T00:00:00.000Z',
+              },
+              {
+                regularProcessDate: '2025-02-01T00:00:00.000Z',
```

Comment:
It looks like the dates come from the server without a time component.

```suggestion
                regularProcessDate: '2025-02-01',
```